template-helpers/articles.js

```diff
@@ -237,7 +237,7 @@ const articleCopy = {
               "Use unique, strong passwords for every account",
               "Use a password manager to remember all your passwords for you",
               "Turn on two-factor authentication for an extra layer of security",
-              "Use a VPN (Virtual Private Network) when using public Wi-Fi",
+              "Use a VPN (like <a target='_blank' href='https://vpn.mozilla.org/?utm_source=monitor.firefox.com&utm_medium=referral&utm_campaign=monitor-security-tips'> Mozilla VPN</a>) when using public Wi-Fi",
```

Comment:
nit: this has a space before " Mozilla VPN", which will cause a semi-awkward, stray underline when hovering over a link.